### PR TITLE
[40k] Implement Neyam Shai Murad

### PR DESCRIPTION
**Pull request by xanderhall**

### CODE REVIEW

**Comment by xanderhall** [comment]
[[Neyam Shai Murad]]

File: Mage.Sets/src/mage/cards/n/NeyamShaiMurad.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        super(card);
+    }
+
+    @Override
+    public NeyamShaiMurad copy() {
+        return new NeyamShaiMurad(this);
+    }
+}
+
+class NeyamShaiMuradTriggeredAbility extends TriggeredAbilityImpl {
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/n/NeyamShaiMurad.java:55]: why not use `DealsCombatDamageToAPlayerTriggeredAbility`?

**Review by xenohedron — COMMENTED** [review]
effect looks fine

File: Mage.Sets/src/mage/cards/n/NeyamShaiMurad.java (hunk 1 above)
Comment (xanderhall) [reply Mage.Sets/src/mage/cards/n/NeyamShaiMurad.java:55]: No idea! Probably a remnant of different iterations of trying to get the effect to work properly. |3

**Review by xenohedron — APPROVED** [review]

**Pull request merged**